src/nft/utils/scrollToTop.ts

```diff
@@ -0,0 +1,3 @@
+const DESKTOP_OFFSET = 420
+
+export const scrollToTop = () => window.scrollTo({ top: DESKTOP_OFFSET })
```

Comment:
I don't disagree, but the reason this is hard coded is because of this:

When we load new assets, number of assets temporarily goes to 0.  For a brief second, it seems that the page is at the top until the assets load again and then at that moment it is no longer at the top. With the addition of the loading assets, we might be able to update this to be dynamic now